### PR TITLE
Setup spacing utility in bootstrap

### PR DESCRIPTION
**Pull request by tingaloo**
I rule and have completed some work on Case Manager that's ready for review!

Bootstrap has a nice inline spacing utility. Instead of creating our utility classes, we can leverage them from BS4. 

I just targeted unique ids and classes that were used once.

Anyways utility CSS is another coffee table discussion because it enforces a structure whilst being highly flexible. The current app has custom utility functions which can use a refactor. But it's definitely worth looking into.

https://getbootstrap.com/docs/4.0/utilities/spacing/

This pull request makes the following changes:
* Setups spacing 
```
    1: ($spacer * .25), //.125 rem
    2: ($spacer * .5), //.25 rem
    3: $spacer, //.5 rem
    4: ($spacer * 1.5), //.75rem
    5: ($spacer * 2), //1rem
    6: ($spacer * 3) //1.5rem
```

It relates to the following issue #s: 
* Bumps #1776

### CODE REVIEW

**Comment by colinxfleming** [comment]
will review a little tighter when we're closer to adding the ab info partial into the working branch, but initial opinion is that I'm way into this!

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ db/*.sqlite3
 /db/*.sqlite3-journal
 /log/*
 /tmp/*
+/bin/*
```
Comment (colinxfleming) [inline .gitignore]: we do wanna keep the `/bin/` directory checked into version control -- these are some key rails-related binaries. are they doing something weird on your machine?

Comment (tingaloo) [reply .gitignore]: woops, i'll put em back.

**Comment by tingaloo** [comment]
lol i named the branch ps4,

**Comment by colinxfleming** [comment]
ps have been laughing every time I look at this because of `ps4`

**Pull request merged**